src/main/java/org/osiam/addons/administration/controller/user/UserViewController.java

```diff
@@ -203,17 +207,95 @@ public String handleLimitAction(@RequestParam(value = REQUEST_PARAMETER_LIMIT) I
+			.addParameter(REQUEST_PARAMETER_OFFSET, session.getPagingInformation().getOffset())
+			.addParameter(REQUEST_PARAMETER_ORDER_BY, session.getPagingInformation().getOrderBy())
+			.addParameter(REQUEST_PARAMETER_ASCENDING, session.getPagingInformation().getAscending())
+			.build();
+	}
+
+	@RequestMapping(params = REQUEST_PARAMETER_ACTION + "=activateSelected",method = RequestMethod.POST)
+	public String handlerUserActivationSelected (
+			@RequestParam(value = REQUEST_PARAMETER_USER_ID_ARRAY) final String[] id_array,
+			@RequestParam(value = REQUEST_PARAMETER_SEND_MAIL) final Boolean sendMail,
+			HttpServletRequest request) {
+
+		User user;
+		for(String user_id : id_array){
+		userService.activateUser(user_id);
+			if (sendMail) {
+				user = userService.getUser(user_id);
+				if(user.getLocale() == null) {
+					emailSender.sendDeactivateMail(user, request.getLocale());
+				} else {
+					emailSender.sendDeactivateMail(user);
+				}
+			}
+		}
+
+		return new RedirectBuilder()
+			.setPath(CONTROLLER_PATH)
+			.addParameter(REQUEST_PARAMETER_ACTIVATE_SELECTED_SUCCESS, true)
+			.addParameter(REQUEST_PARAMETER_QUERY, session.getPagingInformation().getQuery())
+			.addParameter(REQUEST_PARAMETER_LIMIT, session.getPagingInformation().getLimit())
+			.addParameter(REQUEST_PARAMETER_OFFSET, session.getPagingInformation().getOffset())
+			.addParameter(REQUEST_PARAMETER_ORDER_BY, session.getPagingInformation().getOrderBy())
+			.addParameter(REQUEST_PARAMETER_ASCENDING, session.getPagingInformation().getAscending())
+			.build();
+	}
+
+	@RequestMapping(params = REQUEST_PARAMETER_ACTION + "=deleteSelected", method = RequestMethod.POST)
+	public String handleUserDeletionSelected(
+			@RequestParam(value = REQUEST_PARAMETER_USER_ID_ARRAY) final String[] id_array,
+			HttpServletRequest request){
+
+		for(int i=0;i<id_array.length;i++){
+			userService.deleteUser(id_array[i]);
+		}
+
+		return new RedirectBuilder()
+			.setPath(CONTROLLER_PATH)
+			.addParameter(REQUEST_PARAMETER_DELETE_SELECTED_SUCCESS, true)
+			.addParameter(REQUEST_PARAMETER_QUERY, session.getPagingInformation().getQuery())
+			.addParameter(REQUEST_PARAMETER_LIMIT, session.getPagingInformation().getLimit())
+			.addParameter(REQUEST_PARAMETER_OFFSET, session.getPagingInformation().getOffset())
+			.addParameter(REQUEST_PARAMETER_ORDER_BY, session.getPagingInformation().getOrderBy())
+			.addParameter(REQUEST_PARAMETER_ASCENDING, session.getPagingInformation().getAscending())
+			.build();
+	}
+
+	@RequestMapping(params = REQUEST_PARAMETER_ACTION + "=deactivate", method = RequestMethod.POST)
 	public String handleUserDeactivation(
 			@RequestParam(value = REQUEST_PARAMETER_USER_ID) final String id,
 			@RequestParam(value = REQUEST_PARAMETER_SEND_MAIL) final Boolean sendMail,
 			HttpServletRequest request){
 
 		userService.deactivateUser(id);
-
 		if (sendMail) {
 			User user = userService.getUser(id);
-
```

Comment:
please don't delete this empty line. it's good as it is. I like this empty line.